tools/performance/README.md

```diff
@@ -0,0 +1,43 @@
+
+# PowerShell Performance Analysis
+
+This directory contains useful scripts and related files for analyzing PowerShell performance.
+
+If you use the [Windows Performance Toolkit](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk), you can use the following to collect data and analyze a trace.
+
+```PowerShell
+$PowerShellGitRepo = "D:\PowerShell"
+wpr -start $PowerShellGitRepo\tools\performance\PowerShellPerf.wprp -filemode
+pwsh.exe -NoProfile -Command "echo 1"
+wpr -stop PowerShellTrace.etl
+wpa -i wpa://.\PowerShellTrace.etl?profile=$PowerShellGitRepo\tools\performance\PowerShellPerf.wpaProfile
+```
+
+When wpa opens, under System Activity, you'll find a section "Regions of Interest".
+With the above wpaProfile, you should see a bunch of PowerShell related regions as well as GC and JIT activity.
+
+If you use [PerfView](https://github.com/microsoft/perfview), you can collect a trace by running
+
+```PowerShell
+Invoke-PerfviewPS.ps1 -scenario { echo 1 }
+perfview .\perfviewdata.etl
+```
+
+The etl files collected with perfview or wpr should contain roughly the same events.
+
+Also note that you can collect the trace with one tool and analyze with the other.
+
+## Files
+
+| File | Description |
+| ---- | ----------- |
+| GC.wprp | WPA regions of interest for GC |
```

Comment:
These wprp files aren't WPA regions files, they are WPR profiles.

Reply:
Yep, I clearly wasn't thinking when I wrote that - I'm removing the files.